tests/main/interfaces-dbus/task.yaml

```diff
@@ -0,0 +1,83 @@
+summary: Ensure that the dbus interface works
+
+details: |
+    The dbus interface allows owning a name on DBus public bus.
+
+    The test uses two snaps, a provider declaring a dbus slot and a consumer
+    with a plug with the same attributes as the slot. The provider requests
+    a dbus name and, when the plug is connected, the consumer can call the
+    method exposed by the provider.
+
+environment:
+    DISPLAY: :0
+
+systems: [-ubuntu-core-16-*]
+
+prepare: |
+    echo "Give a snap declaring a dbus slot in installed"
+    snap install --edge test-snapd-dbus-provider
+
+    echo "And the D-bus X11 dependencies are installed"
+    apt install -y dbus-x11
+
+    echo "And a snap declaring a matching dbus plug is installed"
+    snap install --edge test-snapd-dbus-consumer
+
+    echo "And the provider dbus loop is started"
+    dbus-launch > dbus.env
```

Comment:
The usual way to do this (as documented in the man-page) is ` eval $(dbus-launch --sh-syntax)` - is this not working for some reason?